lib/main.coffee

```diff
@@ -15,16 +17,16 @@ module.exports =
       default: true
 
   activate: ->
-    require('atom-package-deps').install()
+    require('atom-package-deps').install('linter-pep8')
 
   provideLinter: ->
-    helpers = require('atom-linter')
     provider =
       name: 'pep8'
       grammarScopes: ['source.python']
       scope: 'file' # or 'project'
       lintOnFly: true # must be false for scope: 'project'
       lint: (textEditor)->
+        helpers = require('atom-linter')
```

Comment:
You forgot a question mark here, it should be `helpers ?= require('atom-linter')`